src/Workspaces/Core/MSBuild/Host/Mef/MSBuildMefHostServices.cs

```diff
@@ -2,66 +2,30 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-#nullable disable
-
 using System.Collections.Immutable;
 using System.Reflection;
-using System.Threading;
-using Microsoft.CodeAnalysis.MSBuild.Build;
 
 namespace Microsoft.CodeAnalysis.Host.Mef
 {
     public static class MSBuildMefHostServices
     {
-        private static MefHostServices s_defaultServices;
         public static MefHostServices DefaultServices
         {
             get
             {
-                if (s_defaultServices == null)
-                {
-                    Interlocked.CompareExchange(ref s_defaultServices, MefHostServices.Create(DefaultAssemblies), null);
-                }
-
-                return s_defaultServices;
+                // At this point, we don't have any MEF types in this assembly, so we can just defer to the default set.
+                // This type is just maintained for public API compatibility (and future expansion if we were to have to add types back in.)
+                return MefHostServices.DefaultHost;
```

Comment:
1. API compat with who?
2. would love to consider just reomving this altogether if possible.

Reply:
3. maybe still wrap .DefaultAssemblies (like we did before), that way if that ever changes, this automatically picks it up.  But i don't feel strongly on this.

Reply:
> API compat with who?

Somebody could be make MSBuildWorkspaces by referencing MSBuildMefHostServices.DefaultAssemblies, adding their own additional assemblies and then creating the workspace that way. There's no good way to know if anybody does that (and we can't add telemetry), but even then I can absolutely imagine in the future we'd need to add additional assemblies again. It just happens to be a state of the world _today_ we don't need it, I can't predict the future.